iOS/fastlane/Fastfile

```diff
@@ -80,8 +80,14 @@ platform :ios do
     sync_signing_adhoc(options)
 
     suffix = ""
+    plist_path = "DuckDuckGo/Info.plist"
```

Comment:
>Error Opening Destination: DuckDuckGo/Info.plist

That’s because fastlane works in `fastlane` directory, so you have to prepend it with `../`